dcos-log/api/v1/handlers.go

```diff
@@ -17,6 +17,15 @@ import (
 	"github.com/gorilla/mux"
 )
 
+type key int
+
+var streamKey key = 1
+
+func requestStreamKeyFromContext(ctx context.Context) bool {
```

Comment:
I'm not saying you should change it now, but for future reference. Reviewer first!